truss/constants.py

```diff
@@ -113,9 +113,10 @@
 ]
 AUDIO_MODEL_TRTLLM_REQUIREMENTS = [
     "--extra-index-url https://pypi.nvidia.com",
-    "tensorrt==10.0.1",
-    "tensorrt-cu12==10.0.1",
-    "tensorrt_llm==0.11.0.dev2024061800",
+    "tensorrt_cu12_bindings==10.2.0.post1",
```

Comment:
do you not need the `tensorrt` pin here?

Reply:
they are the same package – the `tensorrt-cu12` is a specifier for the CUDA major version. you can read more about it [here](https://docs.nvidia.com/deeplearning/tensorrt/install-guide/index.html#installing-pip)